doctest/parts/doctest_fwd.h

```diff
@@ -476,6 +476,36 @@ namespace doctest {
 
 DOCTEST_INTERFACE extern bool is_running_in_test;
 
+#if DOCTEST_MSVC
+template <typename T>
+struct doctest_thread_local_wrapper {
```

Comment:
this type is only used in the implementation. There doesn't seem to be a need for it to be in the fwd part. It can be added to doctest.cpp instead.